docs/blog/2019-03-03-new-schema-customization/index.md

```diff
@@ -0,0 +1,270 @@
+---
+title: New schema customization API in Gatsby
+date: 2019-03-03
+author: Mikhail Novikov
+tags: ["schema", "graphql"]
+---
+
+Today we are releasing a preview of a new core Gatsby API - Schema Customization. It gives Gatsby user a much better control over the inferred schema, solving many common issues that people have with their data sources. In addition to adding the new API, we rewrote big chunks of schema generation code from scratch. This gives us a great long-term foundation that will let us make Gatsby GraphQL better in the future.
+
+I would like to thank our community member [Stefan Probst](https://github.com/stefanprobst), who not only did lots of initial groundwork on the refactoring, but also helped immensely with the follow-up work there. We are really happy to have such a great community and super grateful to Stefan for all his hard work. I'd also like to thank [Pavev Chertogov](https://github.com/nodkz/), the author of the [graphql-compose](https://graphql-compose.github.io/) library that we used, who's been super responsive to our bug reports and feature requests.
```

Comment:
Ha thanks for the flowers!

Reply:
![Thank You](https://media.giphy.com/media/nEMmyUp4hl1QOzovKh/giphy.gif)